### DISCUSSION TITLE
New Version 0.19

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#174] [General]
Hi Ladislav,

thank you very much for the new version of QLog and your work on the software. 

The display of QSO duration is well done from my point of view and useful to see how long the QSO already lasts.
For "599" radio operators but rather less interesting, unless they feel animated by the display to lead also once longer CW QSO's, hi.

73, Wolfgang
DL2KI

**Comment by foldynl** [comment]
Hello Wolfgang,

many thanks for your feedback. I'm glad you use QLog. 

You can change a default CW report in `Settings`->`Modes` and double click on CW Report. You can change it to you preferred value of leave it empty.
